qpc_tools/server/ansible/install/roles/server_podman/tasks/start_postgres_container.yml

```diff
@@ -1,13 +1,9 @@
 ---
 
 - name: Start PostgreSQL container with Podman
-  shell: "podman run --name {{ db_name }} --network=container:{{ server_name }} -e POSTGRES_USER={{ db_user }} -e POSTGRES_PASSWORD='{{ db_password }}' -v qpc-data:/var/lib/postgresql/data -d {{db_image_name}}:{{postgres_version}}"
+  shell: "podman run --name {{ db_name }} -e POSTGRES_USER={{ db_user }} -e POSTGRES_PASSWORD='{{ db_password }}' -v qpc-data:/var/lib/postgresql/data -d {{db_image_name}}:{{postgres_version}}"
```

Comment:
Out of curiosity, what was the purpose of `--network=container:{{ server_name }}`?

Reply:
allow communication between quipucords server and db without using just `localhost:<container port>`. It conflicts with the `--ports` and is probably the reason why postgres was running after quipucords. Podman already puts all containers on the same network, so I just removed this and explicitly told qpc container what is the IP for postgres container.